Lesson5/src/app/components/UsersList.js

```diff
@@ -1,37 +1,26 @@
-import React from 'react';
+import React, { useState, useEffect } from "react";
 import axios from 'axios';
 import User from './User';
 
-export default class UsersList extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      users: []
-    };
-  }
+const UsersList = () => {
+  const [userL, setUserL] = useState([]);
 
-  componentDidMount()
-  {
+  useEffect(() => {
     axios.get('https://jsonplaceholder.typicode.com/users/')
-    .then((response) => {
-      this.setState({users: response.data})
-    })
-  }
+      .then((response) => {
+        setUserL(response.data);
+        console.log(userL);
+      })
+  }, []);
 
-  render() {
-    if(!this.state.users.length) {
-      return null;
-    }
+  return (
+    { userL } &&
```

Comment:
```suggestion
    { userL.length } &&
```
так как пустой массив - truthy (переводится в true)